### PR TITLE
Gets dates from status db instead of LIMS

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR has a delivery report that relies on status db for dates instead of LIMS

How to test:
0. install on stage (add instructions on how to as needed)
1. us
1. find/create a family that does exist in status-db but is missing received, prepared, sequenced and delivered dates in LIMS
1. cg upload delivery-report -p [crazyowl]

Expected outcome:
dates for received, prepared, sequenced and delivered exist in the report

### CODE REVIEW

**Comment by ingkebil** [comment]
I'll rebase this branch. It looks like it has the commits I removed from master yesterday

**Comment by ingkebil** [comment]
This looks a lot better :)

File: cg/meta/report/presenter.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+class Presenter:
+    """Contains methods for presentation of different data types"""
+
+    @staticmethod
+    def process_float_string(float_str: str, precision: int) -> str:
+        """Make a float value presentable for the delivery report."""
+        if float_str:
+            presentable_value = str(round(float(float_str), precision))
+        else:
+            presentable_value = 'N/A'
```
Comment (ingkebil) [inline cg/meta/report/presenter.py]: I know ternary operators are to be avoided and the best use case they have is to write assignments like this into one line. Is it more readable to spell it out like it is now? probably.

File: tests/meta/report/test_meta_presenter.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+def test_present_float_string():
+    # GIVEN
+
+    # WHEN formatting None
+    presentable_string = Presenter.process_float_string(None, 0)
+
+    # THEN we should get 'N/A' back
+    assert 'N/A' in presentable_string
```
Comment (ingkebil) [inline tests/meta/report/test_meta_presenter.py]: Shouldn't this just be a `==` comparison?

File: cg/meta/report/api.py [hunk 3]
```diff
@@ -54,23 +57,21 @@ def _get_delivery_data(self, family_id: str) -> dict:
         family_obj = self._get_family_from_status(family_id)
         analysis_obj = family_obj.analyses[0] if family_obj.analyses else None
 
-        report_data['family'] = ReportAPI._present_string(family_obj.name)
+        report_data['family'] = Presenter.process_string(family_obj.name)
```
[the remark this answers is not in the document]
Comment (patrikgrenfeldt) [reply cg/meta/report/api.py]: Thanx :)

File: cg/meta/report/presenter.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/meta/report/presenter.py]: Let's try the ternary one :)

File: cg/meta/report/api.py [hunk 4]
```diff
@@ -256,9 +165,16 @@ def _fetch_family_samples_from_status_db(self, family_id: str) -> list:
...
-            delivery_data_sample['status'] = ReportAPI._present_string(family_sample.status)
-            delivery_data_sample['order_date'] = ReportAPI._present_datetime(sample.ordered_at)
+            delivery_data_sample['ticket'] = Presenter.process_int(sample.ticket_number)
+            delivery_data_sample['status'] = Presenter.process_string(family_sample.status)
+            delivery_data_sample['received'] = Presenter.process_datetime(sample.received_at)
+            delivery_data_sample['prep_date'] = Presenter.process_datetime(sample.prepared_at)
+            delivery_data_sample['sequencing_date'] = Presenter.process_datetime(
+                sample.sequenced_at)
+            delivery_data_sample['delivery_date'] = Presenter.process_datetime(sample.delivered_at)
+            delivery_data_sample['processing_time'] = SampleCalculator.calculate_processing_days(
```
[the remark this answers is not in the document]
Comment (patrikgrenfeldt) [reply cg/meta/report/api.py]: It was handled just fine and I understand that it looks a bit inconsistent in the code so lets change it!

File: tests/meta/report/test_meta_presenter.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply tests/meta/report/test_meta_presenter.py]: done

**Pull request merged**